problem_builder/public/js/instructor_tool.js

```diff
@@ -250,6 +250,88 @@ function InstructorToolBlock(runtime, element) {
         if (statusChanged) updateView();
     }
 
+    // Block types with answers we can export
+    var questionBlockTypes = ['pb-mcq', 'pb-rating', 'pb-answer'];
+
+    function getCourseBlocks() {
+        // Populate course tree options with course blocks
+        $.ajax({
+            type: 'GET',
+            url: $rootBlockId.data('course-blocks-api'),
+            data: {
+                course_id: $element.data('course-id'),
+                requested_fields: 'name,display_name,block_type,children',
+                student_view_data: questionBlockTypes.join(','),
+                all_blocks: true,
+                depth: 'all'
+            },
+            success: updateBlockOptions,
+            dataType: 'json'
+        });
+    }
+    function updateBlockOptions(data) {
+
```

Comment:
@pomegranited Nit: Move empty line up so it separates `getCourseBlocks` and `updateBlockOptions` functions.